### ISSUE TITLE
notebooks images in disconnected enviroment not pullable

### ISSUE THREAD

**Issue by jaysonzhao** [red-hat-data-services/rhods-disconnected-install-helper#34]
Environment:
OpenShift 4.13,  Proxyed to image repo quay.io and registry.redhat.io on Harbor using ImageDigestMirrorSet and ImageTagMirrorSet.

RHODS operator install and instance created normally. 

All IS in ods-applications not imported correctly. It seems the IS is going through API server to pull?
And I am not able to modify the config map for notebook images.

error: tag  failed: Internal error occurred: quay.io/modh/odh-habana-notebooks@sha256:0f6ae8f0b1ef11896336e7f8611e77ccdb992b49a7942bf27e6bc64d73205d05: Get "https://quay.io/v2/": read tcp 192.168.2.27:35204->44.196.174.155:443: read: connection reset by peer
imagestream.image.openshift.io/habana-notebook imported with errors

Name:                   habana-notebook
Namespace:              redhat-ods-applications
Created:                18 minutes ago
Labels:                 app.kubernetes.io/part-of=workbenches
                        app.opendatahub.io/workbenches=true
                        component.opendatahub.io/name=notebooks
                        opendatahub.io/component=true
                        opendatahub.io/notebook-image=true
Annotations:            internal.config.kubernetes.io/previousKinds=ImageStream
                        internal.config.kubernetes.io/previousNames=habana-notebook
                        internal.config.kubernetes.io/previousNamespaces=default
                        opendatahub.io/notebook-image-desc=Jupyter notebook image with a set of habana libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks
                        opendatahub.io/notebook-image-name=HabanaAI
                        opendatahub.io/notebook-image-order=70
                        opendatahub.io/notebook-image-url=https://github.com/red-hat-data-services/notebooks/tree/main/habana
                        opendatahub.io/recommended-accelerators=["habana.ai/gaudi"]
                        openshift.io/image.dockerRepositoryCheck=2023-12-14T03:28:28Z
Image Repository:       image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/habana-notebook
Image Lookup:           local=true
Unique Images:          0
Tags:                   1

2023.2
  tagged from quay.io/modh/odh-habana-notebooks@sha256:0f6ae8f0b1ef11896336e7f8611e77ccdb992b49a7942bf27e6bc64d73205d05

  ! error: Import failed (InternalError): Internal error occurred: quay.io/modh/odh-habana-notebooks@sha256:0f6ae8f0b1ef11896336e7f8611e77ccdb992b49a7942bf27e6bc64d73205d05: Get "https://quay.io/v2/": read tcp 192.168.2.27:35204->44.196.174.155:443: read: connection reset by peer
      Less than a second ago

error: imported completed with errors
[root@cnlvs960 ~]# oc get is
NAME                                IMAGE REPOSITORY                                                                                             TAGS                UPDATED
habana-notebook                     image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/habana-notebook                     2023.2
minimal-gpu                         image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/minimal-gpu                         1.2,2023.1,2023.2
odh-trustyai-notebook               image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/odh-trustyai-notebook               2023.1,2023.2
pytorch                             image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/pytorch                             1.2,2023.1,2023.2
s2i-generic-data-science-notebook   image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-generic-data-science-notebook   1.2,2023.1,2023.2
s2i-minimal-notebook                image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-minimal-notebook                1.2,2023.1,2023.2
tensorflow                          image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/tensorflow                          1.2,2023.1,2023.2

 oc get po -A -owide |grep 192.168.2.27
openshift-apiserver                                apiserver-6d8fb57d5f-hwgp4                                                 2/2     Running     0               34m     192.168.2.27    cnocpmaster3.xxxxxx.xxxxxxx.intranet.local   <none>           <none>
[root@cnlvs960 ~]#

**Comment by zdtsw** [comment]
@jaysonzhao did you get it passed or this still blocking your upgrade?
